docs/en/observability/synthetics-quickstart.asciidoc

```diff
@@ -139,7 +139,7 @@ Don't forget to update the example with your {es} credentials.
 
```

Comment:
I'd like to add a note so other maintainers know why we're using `{minor-version}.0` here

```suggestion

// Right now we do not publish docker images for patch releases
// Docker pull should reference only the .0 bugfix of the minor
// We can accomplish this with the following attribute `{minor-version}.0`

```